Mathlib/Order/OmegaCompletePartialOrder.lean

```diff
@@ -392,11 +459,27 @@ namespace OmegaCompletePartialOrder
 
 variable [∀ x, OmegaCompletePartialOrder <| β x]
 variable [OmegaCompletePartialOrder γ]
+variable {f : γ → ∀ x, β x}
```

Comment:
Seems like there's a missing lemma saying that for `f : a -> b -> c`, `ωScottContinuous f` implies `ωScottContinuous (f a)`

Reply:
@b-mehta sorry, I can't quite see what the statement would be. Please can you elaborate? Thanks.